libledger/LedgerParam.cpp

```diff
@@ -689,5 +690,36 @@ void LedgerParam::initFlowControlConfig(boost::property_tree::ptree const& _pt)
                                  mutableFlowControlParam().outGoingBandwidthLimit);
 }
 
+void LedgerParam::initNodeListBySection(dev::h512s& _nodeList,

```

Comment:
confused about this function, not used?

Reply:
I think this function maybe used multi-time in the future versions.

Reply:
parseListOfSection？

Reply:
Done.